wolfssl/wolfcrypt/blake2-int.h

```diff
@@ -73,7 +73,7 @@
     byte  node_depth;    /* 15 */
     byte  inner_length;  /* 16 */
     /* byte  reserved[0]; */
-    byte  salt[BLAKE2B_SALTBYTES]; /* 24 */
+    byte  salt[BLAKE2S_SALTBYTES]; /* 24 */
```

Comment:
Thanks for fixing the copy/paste error here... which used the wrong salt size for blake2b.... 16 vs 24.